core/page-assets/loader.php

```diff
@@ -35,6 +35,11 @@ private function init_assets() {
 					'version' => ELEMENTOR_VERSION,
 					'dependencies' => [],
 				],
+				'e-apple-webkit' => [
+					'src' => $this->get_css_assets_url( 'appleWebkit', 'assets/css/', true ),
+					'version' => ELEMENTOR_VERSION,
```

Comment:
@rami-elementor Is this definitely the correct place to init these assets?

Reply:
Register inside includes/frontend.php.